js/engine/Main.ml

```diff
@@ -48,15 +48,9 @@ let _ =
            in
            let targets =
              List.map
-               (fun f ->
-                 `CodeTarget
-                   Input_to_core_t.
-                     {
-                       path = f;
-                       analyzer = xlang;
-                       products = Product.all;
-                       lockfile_target = None;
-                     })
+               (fun f : Target.t ->
+                 Code
+                   (Target.code_of_origin xlang Product.all (File (Fpath.v f))))
```

Comment:
code_of_origin is a bad name given you don't pass just an origin here but lots of arguments.

Reply:
Done in 1e38a3c65ba5f117024690f171ed62c91a1ad09f. I have called this functions`mk_code` instead (and likewise for the other versions). Please let me know a more concrete suggestion if that's also disliked.